pypsa_to_IPCC.py

```diff
@@ -293,14 +403,108 @@
             var['OM Cost|Fixed|Hydrogen|Electricity'] = factor * 0.01*costs.loc[('electrolysis', 'FOM'),'value']*costs.loc[('electrolysis', 'investment'),'value']
             var['OM Cost|Fixed|Electricity|Biomass|w/o CCS'] = factor * 0.01*costs.loc[('biomass EOP', 'FOM'),'value']*costs.loc[('biomass EOP', 'investment'),'value']
 
+            """
+            Investment
+            """
+            var['Investment|Energy Supply|Electricity|Biomass|w/ CCS'] = (n.links.p_nom_opt.filter(like='biomass CHP CC').filter(like=str(year)).filter(like=country)*
+                                         n.links.capital_cost.filter(like='biomass CHP CC').filter(like=str(year)).filter(like=country)).sum()
+            var['Investment|Energy Supply|Electricity|Biomass|w/o CCS'] = (n.links.p_nom_opt.filter(like='biomass CHP').filter(like=str(year)).filter(like=country)*
+                                         n.links.capital_cost.filter(like='biomass CHP').filter(like=str(year)).filter(like=country)).sum()
+            var['Investment|Energy Supply|Electricity|Coal|w/ CCS'] = (n.links.p_nom_opt.filter(like='coal CC').filter(like=str(year)).filter(like=country)*
+                                         n.links.capital_cost.filter(like='coal CC').filter(like=str(year)).filter(like=country)).sum()
+            var['Investment|Energy Supply|Electricity|Coal|w/o CCS'] = (n.links.p_nom_opt.filter(like='coal').filter(like=str(year)).filter(like=country)*
+                                         n.links.capital_cost.filter(like='coal').filter(like=str(year)).filter(like=country)).sum()
+            var['Investment|Energy Supply|Electricity|Electricity Storage'] = (n.links.p_nom_opt.filter(like='coal').filter(like=str(year)).filter(like=country)*
+                                         n.links.capital_cost.filter(like='coal').filter(like=str(year)).filter(like=country)).sum()
+            var['Investment|Energy Supply|Electricity|Fossil'] = (n.links.p_nom_opt.filter(like='coal').filter(like=str(year)).filter(like=country)*
+                                         n.links.capital_cost.filter(like='coal').filter(like=str(year)).filter(like=country)).sum()
```

Comment:
Should the fossil investments only include coal investments?

Reply:
Also fixed. Now includes gas (CCGT and OCGT) and coal, but maybe I'll somehow add CHPs later as well.